File: java/BEE1020.java
import java.util.Scanner;

public class BEE1020 {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        int n, dias, meses, anos;

        // Entrada
        n = teclado.nextInt();

        // Processamento
        anos = n / 365;
        n = n % 365;

        meses = n / 30;
        dias = n % 30;

        // Saída
        System.out.println(anos + " ano(s)");
        System.out.println(meses + " mes(es)");
        System.out.println(dias + " dia(s)");

        teclado.close();
    }
}

File: python/BEE1020.py
# -*- coding: utf-8 -*-

"""
Escreva a sua solução aqui
Code your solutioentrada here
Escriba su solucióentrada aquí
"""

entrada = int(input())

anos = int((entrada - (entrada % 365)) / 365)
entrada = entrada - 365 * anos
meses = int((entrada - (entrada % 30)) / 30)
entrada = entrada - 30 * meses
dias = int(entrada)

print(f"{anos} ano(s)")
print(f"{meses} mes(es)")
print(f"{dias} dia(s)")
